test/testdata/expected/EmptyStruct.go

```diff
@@ -0,0 +1,7 @@
+package expected
```

Comment:
It is a Go convention to put all test data in the "testdata" folder. So please move all testdata into $root/testdata